proposals/NNNN-vehicle-type-filter.md

```diff
@@ -0,0 +1,435 @@
+            <key>model</key>
+            <string>Mustang</string>
+            <key>modelYear</key>
+            <string>2019</string>
+            <key>trim</key>
+            <string>GT</string>
+        </dict>
+        <dict>
+            <key>make</key>
+            <string>OEM1</string>
+            <key>model</key>
+            <string>OEM model</string>
+            <key>modelYear</key>
+            <string>2019</string>
+            <key>trim</key>
+            <string>trim info</string>
+        </dict>
+    </array>
+```
+```xml
+<key>SDLSupportedVehicleTypes</key>
+    <array>
+        <dict>
+            <key>make</key>
+            <string>Ford</string>
+            <key>model</key>
+            <string>Mustang</string>
+        </dict>
+    </array>
+```
+```xml
+<key>SDLSupportedVehicleTypes</key>
+    <array>
+        <dict>
+            <key>make</key>
+            <string>Ford</string>
+            <key>model</key>
+            <string>Mustang</string>
+            <key>modelYear</key>
+            <string>2019</string>
+        </dict>
+        <dict>
+            <key>make</key>
+            <string>OEM1</string>
+            <key>model</key>
+            <string>OEM model</string>
+            <key>modelYear</key>
+            <string>2019</string>
+        </dict>
+    </array>
+```
+
+```xml
+<key>SDLSupportedVehicleTypes</key>
+    <array>
+        <dict>
+            <key>make</key>
+            <string>Ford</string>
+            <key>model</key>
+            <string>Mustang</string>
+            <key>trim</key>
+            <string>GT</string>
+        </dict>
+        <dict>
+            <key>make</key>
+            <string>OEM1</string>
+            <key>model</key>
+            <string>OEM model</string>
+            <key>trim</key>
+            <string>trim info</string>
+        </dict>
+    </array>
+```
+```xml
+<key>SDLSupportedVehicleTypes</key>
+    <array>
+        <dict>
+            <key>make</key>
+            <string>Ford</string>
+        </dict>
+        <dict>
+            <key>make</key>
+            <string>OEM1</string>
+        </dict>
+    </array>
+```
+#### Determine Vehicle type info
+
+1. After version negotiations and starting RPC service, the proxy should check if `Get vehicle type` is supported by the SDL enabled IVI system.
+2. If the protocol version supports `Get vehicle type`, the proxy will send the `Get vehicle type` protocol message.
+3. On receiving vehicle type information, proxy will check `SDLSupportedVehicleTypes` keys to check supported vehicle type.
+4. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. The iOS proxy can implement `OnSDLEnabled` notification similar to that of an android proxy to notify the app about SDL connection with the supported vehicle. The app can provide life cycle configuration to the proxy on receiving `OnSDLEnabled` notification. 
```

Comment:
```suggestion
4. If the vehicle type is supported, the proxy should also notify the app about the connected vehicle type so that the app can configure SDL as required. The iOS proxy can implement an `OnSDLEnabled` notification similar to that of the Android proxy to notify the app about SDL connection with the supported vehicle. The app can provide lifecycle configuration to the proxy upon receiving the `OnSDLEnabled` notification. 
```